app/models/accounting/quickbooks/account_fetcher.rb

```diff
@@ -0,0 +1,35 @@
+module Accounting
+  module Quickbooks
+    class AccountFetcher
```

Comment:
There is quite a bit of repeated code here and in TransactionFetcher. Could you refactor to use inheritance?